docs/finality-provider-operation.md

```diff
@@ -0,0 +1,732 @@
+transactions, such as block votes and public randomness submissions that are 
+essential for its liveness and earning of rewards. 
+
+For the `fpd` keyring, the `test` backend will be exclusively used, and it is 
+mandatory that you  follow this practice until automated key management becomes 
+available. Additionally, we are also exploring options to support different 
+withdrawal addresses, so that rewards can go to a separate address.
+
+It is also important to note that the finality provider daemon will refund 
+fees for the submission of valid votes as those are essential for the protocol.
+All other transactions, will require gas, but will be happening infrequently
+or only once. As this keyring is used for both earning and 
+operational purposes, we strongly recommend maintaining only the necessary 
+funds for operations in the keyring, and extracting the rest into
+more secure locations.
+
+> ⚠️ **Important**:
+> To operate your Finality Provider, ensure your Babylon account is funded. 
+> Block vote transactions have their gas fees refunded, but public randomness 
+> submissions require gas payments. For testnet, you can obtain funds from our 
+> [faucet](#add-faucet).
+<!-- TODO: add faucet link -->
+
+Use the following command to add the Babylon key for your finality provider:
+
+```shell 
+fpd keys add <key-name> --keyring-backend test --home <path>
+```
+
+The above `keys add` command will create a new key pair and store it in your keyring. 
+The output should look similar to the one below:
+
+``` json
+{
+  "address": "bbn19gulf0a4yz87twpjl8cxnerc2wr2xqm9fsygn9",
+  "name": "finality-provider",
+  "pubkey": {
+    "@type": "/cosmos.crypto.secp256k1.PubKey",
+    "key": "AhZAL00gKplLQKpLMiXPBqaKCoiessoewOaEATKd4Rcy"
+  },
+  "type": "local"
+}
+```
+
+This command will automatically update the `Key` field in 
+the config file to use this key name. This key will be used for all interactions 
+with the Babylon chain, including finality provider registration and transaction 
+signing.
+ 
+### 4.3. Configure Your Finality Provider
+
+Edit the `fpd.conf` file in your finality provider home directory with the 
+following parameters:
+
+```shell 
+[Application Options] 
+EOTSManagerAddress = 127.0.0.1:12582 
+RpcListener = 127.0.0.1:12581
+
+[babylon] 
+Key = <finality-provider-key-name-signer> # the key you used above
+ChainID = bbn-test-5 # chain ID of the Babylon chain
+RPCAddr = http://127.0.0.1:26657 
+GRPCAddr = https://127.0.0.1:9090 
+KeyDirectory = <path> # The `--home`path to the directory where the keyring is stored
+``` 
+
+> ⚠️ **Important**: Operating a finality provider requires a connection to a 
+> Babylon blockchain node. It is **highly recommended** to operate your own 
+> Babylon full node instead of relying on third parties. You can find 
+> instructions on setting up a Babylon node 
+> [here](https://github.com/babylonlabs-io/networks/tree/main/bbn-1/node-setup).
```

Comment:
Do we want to update the link before launch as it points to `bbn-1`? If so, better to add a TODO here